src/esp/physics/bullet/BulletURDFImporter.cpp

```diff
@@ -570,33 +570,6 @@ Mn::Matrix4 BulletURDFImporter::ConvertURDF2BulletInternal(
       bool isDynamic =
           (mbLinkIndex < 0 && cache.m_bulletMultiBody->hasFixedBase()) ? false
                                                                        : true;
-      int collisionFilterGroup = isDynamic
```

Comment:
This block is being moved below. Beware I'm also changing the collision filtering in the new version below.